src/Elliptic/Systems/ScalarGaussBonnet/FirstOrderSystem.hpp

```diff
@@ -0,0 +1,51 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "Elliptic/BoundaryConditions/BoundaryCondition.hpp"
+#include "Elliptic/Protocols/FirstOrderSystem.hpp"
+#include "Elliptic/Systems/ScalarGaussBonnet/Equations.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace sgb {
+
+/*!
+ * \brief The scalar equation in sGB theories of gravity in the decoupled limit,
+ * assuming quasi-stationarity. See \ref sgb for details on the
+ * explicit equation.
+ */
+struct FirstOrderSystem
+    : tt::ConformsTo<elliptic::protocols::FirstOrderSystem> {
+ private:
+  using field = ::sgb::Tags::Psi;
+
+ public:
+  static constexpr size_t volume_dim = 3;
+
+  using primal_fields = tmpl::list<field>;
+  using primal_fluxes =
+      tmpl::list<::Tags::Flux<field, tmpl::size_t<3>, Frame::Inertial>>;
+
+  using background_fields = tmpl::list<
```

Comment:
Please add a comment that numerically imported background fields are not listed here